desc/basis.py

```diff
@@ -186,6 +187,11 @@ def NFP(self):
         """int: Number of field periods."""
         return self.__dict__.setdefault("_NFP", 1)
 
+    @property
+    def NFP_umbilic_factor(self):
+        """int: Umbilic factor field periods."""
+        return self.__dict__.setdefault("_NFP_umbilic_factor", 1)
```

Comment:
You may need to add logic to [def _set_up](https://github.com/PlasmaControl/DESC/blob/363826c3ee0cbb71f4a0d92966ccedbe6b20a24e/desc/basis.py#L50) to set this factor for old equilibrium. Sometimes it might be as simple as
`self._NFP_umbilic_factor = self.NFP_umbilic_factor`

The setup method will be called automatically when you load an old file with examples.get("W7-X"). (The code that does this is here https://github.com/PlasmaControl/DESC/blob/4745460ede4ad2e3e9d5d25c762e1f963b4ce34a/desc/io/optimizable_io.py#L75)